diff --git src/Driver/DiscordClient.php src/Driver/DiscordClient.php
index 7607aca..3c31d0e 100644
--- src/Driver/DiscordClient.php
+++ src/Driver/DiscordClient.php
@@ -285,6 +285,14 @@ class DiscordClient implements IClient
         } while (true);
     }
 
+    CONST whitelistRoleIds = [
+        "594654571373264906", //Fleets
+        "578296482990522368", //Zero To High.
+        "631959946308616245", //Zero To High. Capitals
+        "631960389202083973", //Zero To High. Wormhole
+        "594654873220415488" //Fleet Com.
+    ];
+
     /**
      * @throws \GuzzleHttp\Exception\GuzzleException
      * @throws \Seat\Services\Exceptions\SettingException
@@ -302,6 +310,8 @@ class DiscordClient implements IClient
             // ignore managed roles (ie: booster)
             if ($role_attributes['managed']) continue;
 
+            if (!in_array($role_attributes['id'], self::whitelistRoleIds)) continue;
+
             $role = new DiscordRole($role_attributes);
             $this->roles->put($role->getId(), $role);
         }
diff --git src/Driver/DiscordMember.php src/Driver/DiscordMember.php
index a610e2d..128182f 100644
--- src/Driver/DiscordMember.php
+++ src/Driver/DiscordMember.php
@@ -101,6 +101,7 @@ class DiscordMember implements IUser
      */
     public function setName(string $name): bool
     {
+        /*
         $nickname = Str::limit($name, 32, '');
 
         DiscordClient::getInstance()->sendCall('PATCH', '/guilds/{guild.id}/members/{user.id}', [
@@ -110,8 +111,9 @@ class DiscordMember implements IUser
         ]);
 
         $this->nick = $nickname;
+        */
 
-        return true;
+        return false;
     }
 
     /**
