diff --git src/Driver/DiscordClient.php src/Driver/DiscordClient.php
index 7607aca..cd29026 100644
--- src/Driver/DiscordClient.php
+++ src/Driver/DiscordClient.php
@@ -285,6 +285,13 @@ class DiscordClient implements IClient
         } while (true);
     }
 
+    CONST whitelistRoleIds = [
+        "630206940127035425", //Fleets
+        "630206902185492510", //Zero To High.
+        "631168195360849931", //Zero To High. Capitals
+        "631166255541256232" //Fleet Com.
+    ];
+
     /**
      * @throws \GuzzleHttp\Exception\GuzzleException
      * @throws \Seat\Services\Exceptions\SettingException
@@ -302,6 +309,8 @@ class DiscordClient implements IClient
             // ignore managed roles (ie: booster)
             if ($role_attributes['managed']) continue;
 
+            if (!in_array($role_attributes['id'], self::whitelistRoleIds)) continue;
+
             $role = new DiscordRole($role_attributes);
             $this->roles->put($role->getId(), $role);
         }
diff --git src/Driver/DiscordMember.php src/Driver/DiscordMember.php
index a610e2d..e57c758 100644
--- src/Driver/DiscordMember.php
+++ src/Driver/DiscordMember.php
@@ -101,6 +101,7 @@ class DiscordMember implements IUser
      */
     public function setName(string $name): bool
     {
+        /*
         $nickname = Str::limit($name, 32, '');
 
         DiscordClient::getInstance()->sendCall('PATCH', '/guilds/{guild.id}/members/{user.id}', [
@@ -110,6 +111,7 @@ class DiscordMember implements IUser
         ]);
 
         $this->nick = $nickname;
+        */
 
         return true;
     }
